sdk/core/core-paging/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@azure/azure-sdk/ts-package-json-sideeffects": "off",
+    "@azure/azure-sdk/ts-package-json-main-is-cjs": "off",
+    "@azure/azure-sdk/ts-package-json-module": "off",
+    "@azure/azure-sdk/ts-package-json-files-required": "off"
```

Comment:
Generally we turn off rules because we don't want to invest in fixing things right this moment, but it is open for fixing it in the future. In this case, since this package is different from others, these rules **have** to be turned off. Can we add comments stating why these are turned off and mention the same in the PR description?

Reply:
@ramya-rao-a fixed.